mne_connectivity/spectral/tests/test_spectral.py

```diff
@@ -546,25 +553,18 @@ def test_spectral_connectivity_epochs_spectrum_input(method, mode):
     freqs_noise = (freqs < fband[0] - trans_bandwidth * 2) | (
         freqs > fband[1] + trans_bandwidth * 2
     )
-
-    # nothing for CaCoh to optimise, so use same thresholds for CaCoh and Coh
-    if mode == "multitaper":  # lower baseline for multitaper
-        con_thresh = (0.1, 0.3)
-    else:  # higher baseline for Welch/Fourier
-        con_thresh = (0.2, 0.4)
-
     # check freqs of simulated interaction show strong connectivity
-    assert_array_less(con_thresh[1], np.abs(con.get_data()[:, freqs_con].mean()))
+    assert_array_less(0.6, np.abs(con.get_data()[:, freqs_con].mean()))
     # check freqs of no simulated interaction (just noise) show weak connectivity
-    assert_array_less(np.abs(con.get_data()[:, freqs_noise].mean()), con_thresh[0])
+    assert_array_less(np.abs(con.get_data()[:, freqs_noise].mean()), 0.3)
 
 
 # TODO: Add general test for error catching for spec_conn_epochs
 @pytest.mark.skipif(
     not check_version("mne", "1.8"), reason="Requires MNE v1.8.0 or higher"
 )
-def test_spectral_connectivity_epochs_spectrum_input_error_catch():
-    """Test spec_conn_epochs catches error with EpochsSpectrum data as input."""
+def test_spectral_connectivity_epochs_spectrum_tfr_input_error_catch():
+    """Test spec_conn_epochs catches errors with EpochsSpectrum/TFR data as input."""
```

Comment:
why change the test name if not changing the function to actually test TFR input?